Makefile

```diff
@@ -25,7 +25,8 @@ export CGO_CFLAGS
 TOOLS = \
 	github.com/fzipp/gocyclo \
 	gitlab.com/opennota/check/cmd/varcheck \
-	github.com/mendersoftware/deadcode
+	github.com/mendersoftware/deadcode \
+	github.com/oleorhagen/gobinarycoverage
```

Comment:
Should change this to `mendersoftware` before merging.

Reply:
Alright, that means we should probably at least polish the `Readme` a bit more first.

Reply:
Yes, although I thought it was already pretty good. I don't consider this tool a slick, sexy, and user friendly experience. More like the dirty, but trusty tool which won't let you down, by developers, for developers! :-)

Reply:
haha, no. Definitely not an Apple product :p